Attach the new window at the bottom of the stack

diff --git a/dwl.c b/dwl.c
index b09fc6f..df4fe05 100644
--- a/dwl.c
+++ b/dwl.c
@@ -1366,7 +1366,10 @@ mapnotify(struct wl_listener *listener, void *data)
 	c->geom.height += 2 * c->bw;
 
 	/* Insert this client into client lists. */
-	wl_list_insert(&clients, &c->link);
+   if (clients.prev)
+       wl_list_insert(clients.prev, &c->link); /* tile at bottom */
+   else
+       wl_list_insert(&clients, &c->link);
 	wl_list_insert(&fstack, &c->flink);
 
 	/* Set initial monitor, tags, floating status, and focus */
